mifospay/src/main/java/org/mifos/mobilewallet/mifospay/history/ui/HistoryFragment.java

```diff
@@ -87,6 +92,29 @@ private void setupUi() {
         vStateView.setVisibility(View.GONE);
     }
 
+    @Override
+    public void scrollDownBottom() {
+        rvHistory.addOnScrollListener(new RecyclerView.OnScrollListener() {
+            @Override
+            public void onScrollStateChanged(RecyclerView recyclerView, int newState) {
+                super.onScrollStateChanged(recyclerView, newState);
+                if (!recyclerView.canScrollVertically(1)) {
+                    Toast.makeText(getContext(), R.string.end_of_transactions,
+                            Toast.LENGTH_SHORT).show();
+                } else {
+                    scrollDownButton.setVisibility(View.VISIBLE);
```

Comment:
Since you have written this line I guess you intended to hide the button when the user is at the bottom of the RecyclerView. 
If that's the case make sure you hide the button in the if block.

Reply:
You're right, I have not hidden it. Thank you for the heads up!!
Shall do it and squash